src/hooks/useMixpanel.ts

```diff
@@ -607,7 +608,7 @@ export default function useMixpanel(trade?: Aggregator | undefined, currencies?:
             break
           mixpanelHandler(MIXPANEL_TYPE.SWAP_COMPLETED, {
             arbitrary: transaction.arbitrary,
-            actual_gas: transaction.receipt?.gasUsed || '',
+            actual_gas: transaction.receipt?.gasUsed || BigNumber.from(0),
```

Comment:
this will fix to_number error, right? 
very nice :D

Reply:
yeah, my bad later on